### DISCUSSION TITLE
Maven artifactId conventions

### DISCUSSION THREAD

**Discussion by reismade** [eclipse-birt/birt#1073] [General]
Hi,

I'm migrating an application that's using an old version of birt from Nexus to Artifactory, but Artifactory complains about the artifactId not conforming with the standard naming conventions and thus doesn't allow me to upload it.
According to https://maven.apache.org/guides/mini/guide-naming-conventions.html, the artifactId is
"the name of the jar without version. If you created it, then you can choose whatever name you want with lowercase letters and no strange symbols. If it's a third party jar, you have to take the name of the jar as it's distributed, eg. maven, commons-math"
I came here to check if the latest version still follows the old artifactId, and unfortunately, it does: <artifactId>org.eclipse.birt-parent</artifactId>
I've always wondered why you'd want to use the groupId in the artifactId, since it's just redundant information at best and contradictory in worst case.
Is there any chance you could change that?

Best regards,
Mario

**Comment by wimjongman** [comment]
Sure, we are not married to that. Your remarks make perfect sense. Can you supply a patch?

**Reply by reismade** [reply]
I tried to make sense of the whole pom structure and change it, but I'm not a developer. You have 5 levels of child poms, and on some levels there are both child projects with and without maven-compliant artifactIds. I feel like this change requires a deeper understanding of the project structure.
